hack/tests/integration.sh

```diff
@@ -4,7 +4,7 @@ set -eu
 
 source hack/lib/image_lib.sh
 
-export OSDK_INTEGRATION_IMAGE="quay.io/example/memcached-operator:v0.0.2"
+export OSDK_INTEGRATION_IMAGE="quay.io/example/memcached-operator:latest"
```

Comment:
why we changed to latest?

Reply:
The tag doesn't matter since the image is used however the test wants to use it. I just made it more general.